src/query/storages/common/table_meta/src/meta/utils.rs

```diff
@@ -47,3 +55,24 @@ pub fn monotonically_increased_timestamp(
     }
     timestamp
 }
+
+pub fn parse_storage_prefix(options: &BTreeMap<String, String>, table_id: u64) -> Result<String> {
+    // if OPT_KE_STORAGE_PREFIX is specified, use it as storage prefix
+    if let Some(prefix) = options.get(OPT_KEY_STORAGE_PREFIX) {
+        return Ok(prefix.clone());
+    }
```

Comment:
Is stmt that creates temp table with external location prohibited?
, e.g. `create temp table t (..) 's3://somebuck/..`.

Just a reminder, we can address this in a dedicated PR (if necessary)

Reply:
oh... please ignore this, `OPT_KEY_STORAGE_PREFIX` is for `attache table`, nothing to do with create table with external location